cirq-google/cirq_google/engine/engine_sampler.py

```diff
@@ -49,26 +51,28 @@ def __init__(
         self._processor_ids = [processor_id] if isinstance(processor_id, str) else processor_id
         self._engine = engine
 
-    def run_sweep(
+    async def run_sweep_async(
```

Comment:
following #5361 #5432 QuantumEngineSampler is deprecated. These changes need to be applied to `ProcessorSampler`. Maybe don't bother adding them here?

Reply:
Reverted changes to `engine_sampler.py`